src/server/key_ttl_compaction_filter.h

```diff
@@ -60,6 +62,23 @@ class KeyWithTTLCompactionFilter : public rocksdb::CompactionFilter
             return false;
         }
 
+        // ignore empty write
+        if (key.size() < 2) {
```

Comment:
if user's key < 2 actually?

Reply:
In the compaction operation,  It will extract field from key. If key.size() < 2, it will cause coredump